test/math/FloatVectorListTest.ooc

```diff
@@ -123,7 +123,8 @@ FloatVectorListTest: class extends Fixture {
 			list add(2.0f)
 			list add(3.0f)
 			list add(4.0f)
-			slice := list getSlice(1 .. 2)
+			slice := list getSlice(1, 2)
```

Comment:
OK, if you keep this line, and fix the `getSlice` function above like I said, this call will "test" both versions, which is nice.